### PR TITLE
add feature - generate random problem

### PR DESCRIPTION
**Pull request by yue2388253**
use "cargo run random" to generate a random problem.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -16,8 +16,18 @@ fn main() {
...
         panic!("problem id must be provided");
     }
-    let id = &args[1];
-    let id = id.parse::<u32>().expect(&format!("not a number: {}", id));
+    let id_arg = &args[1];
+    let mut id :u32 = 0;
+    match id_arg.as_ref() {
+        "random" => {
+            println!("You select random mode.");
+            id = get_random_id();
```
Comment (aylei) [inline src/main.rs]: What if the problem is locked or don't have a rust version?

**Review by aylei — COMMENTED** [review]
Rest LGTM.
And this idea is interesting 😜

File: src/main.rs (hunk 1 above)
Comment (yue2388253) [reply src/main.rs]: yes, current implementation has such limitation. 
Do you think we should verify whether the problem of generated id meet these two requirement at first? And regenerate an id that meets these two requirement

Comment (yue2388253) [reply src/main.rs]: In addition, using random mode to generate problems results in the disorder of lib.rs

Comment (aylei) [reply src/main.rs]: 1. I think it is nice to have the ID re-generation, but a proper error message is also okay.
2. It's not a problem, we don't have an order guarantee for `lib.rs`

**Comment by yue2388253** [comment]
Now that the program can handle the case that the problem has no rust version.

However, I don't know what "the problem is locked" means? Could you please say it in more details?

**Comment by aylei** [comment]
> Now that the program can handle the case that the problem has no rust version.
> 
> However, I don't know what "the problem is locked" means? Could you please say it in more details?

I mean some problems are only available for leetcode premium, the subscription plan

**Review by aylei — COMMENTED** [review]
LGTM, is this PR ready to merge? @yue2388253

**Review by aylei — APPROVED** [review]

**Comment by yue2388253** [comment]
Sorry to late reply. This PR is ready for merge.

BTW, recently I am preparing my paper and thus might not be able to work on this repo.

**Comment by aylei** [comment]
@yue2388253 Thanks!

**Pull request merged**